### PR TITLE
Server models 2024-05-31

### PR DESCRIPTION
**Pull request by iOvergaard**
## Description

Generate new models and add `template` to the blueprint model. I am a little unsure of this since the `template` is not returned from the server in the details request anyway.

### CODE REVIEW

File: src/packages/documents/document-blueprints/repository/detail/document-blueprint-detail.server.data-source.ts [hunk 1]
```diff
@@ -133,7 +135,7 @@ export class UmbDocumentBlueprintServerDataSource implements UmbDetailDataSource
 			id: model.unique,
```
Comment (madsrasmussen) [inline src/packages/documents/document-blueprints/repository/detail/document-blueprint-detail.server.data-source.ts:135]: Is this the correct create model? Should it be 
`CreateDocumentBlueprintRequestModel` ?

Comment (iOvergaard) [reply src/packages/documents/document-blueprints/repository/detail/document-blueprint-detail.server.data-source.ts:135]: Ohh, didn't notice that before. You may be right, let me see.

Comment (iOvergaard) [reply src/packages/documents/document-blueprints/repository/detail/document-blueprint-detail.server.data-source.ts:133]: Found the same problem with the update model (`UpdateDocumentBlueprintRequestModel`), albeit that didn't require a template. But now we use the correct models at least.

**Review by madsrasmussen — APPROVED** [review]
I wonder way the typescript didn't catch that we were passing the wrong model 🤔

**Comment by iOvergaard** [comment]
> I wonder why typescript didn't catch that we were passing the wrong model 🤔

The two models were identical up until yesterday 💯

**Pull request merged**